### ISSUE TITLE
Logout button - include js.cookie.js script

### ISSUE THREAD

**Issue by tstavinoha** [spring-guides/tut-spring-boot-oauth2#63]
js-cookie needs to be imported in head tag before it can be used.

```
<head>
    ....
    <script type="text/javascript" src="/webjars/js-cookie/js.cookie.js"></script>
</head>
```

Otherwise, the following error shows up in the developer console:

```Uncaught ReferenceError: Cookies is not defined
    at Object.beforeSend (http://localhost:8080/:41:25)
    at Function.ajax (http://localhost:8080/webjars/jquery/jquery.min.js:4:10323)
    at Function.n.(anonymous function) [as post] (http://localhost:8080/webjars/jquery/jquery.min.js:4:11574)
    at logout (http://localhost:8080/:54:11)
    at HTMLButtonElement.onclick (http://localhost:8080/#_=_:26:64)

**Comment by dsyer** [comment]
Works for me (it's already included at line 27: https://github.com/spring-guides/tut-spring-boot-oauth2/blob/6fb13971823c2de638a1e133f4f5ac12361cf065/logout/src/main/resources/static/index.html#L27).

**Comment by tstavinoha** [comment]
Yeah, I've seen it a bit later. I created a PR to just put this step in the tutorial documentation, because all the other changes were explicitly listed and guide was smooth until that point.

**Comment by dsyer** [comment]
Fixed in #64
